### DISCUSSION TITLE
Content overview plugin - get a more focused view on your content

### DISCUSSION THREAD

**Discussion by wsydney76** [craftcms/cms#12280] [Plugin ideas]
Just a litle internal plugin that came to life today, but may be it is of interest to someone:

https://github.com/wsydney76/craft-contentoverview

It was created out of the desire to get a more condensed overview of current entries, especially those currently in progress.

It would be interesting to release this as an official plugin, unfortunately there is no capacity for it at the moment, so I'm just mentioning it here. Free for adoption....

* Show different sections in different layouts (cards, cardlets, list, line). Add section specific infos and image:

![screenshot1](https://user-images.githubusercontent.com/60010597/200640376-b70428ba-7c6b-4b0a-bd89-8a4f65688cd4.jpg)

* Support your workflow and quality management: Show drafts / status / own provisional drafts

![screenshot2](https://user-images.githubusercontent.com/60010597/200640466-094ca396-314b-4f5b-a323-be61bf9d688a.jpg)

Heavily inspired by Kirby CMS.

/Werner et al.

**Comment by thupsi** [comment]
Nice! Let's see if the 'unified content view' in Craft 5 will offer similar functionality.

**Reply by wsydney76** [reply]
Hopefully, will need a lot of customization options though.

When we began to evaluate this idea in a real-life project, things escalated quickly (oh, I have another idea!) and the simple 'overview' turned into something like a 'content management dashboard', see [this example](https://github.com/wsydney76/craft-contentoverview#customization-example)

People like the idea of getting a more streamlined experience, visually appealing, simple to understand, focused on their current work, respecting the internal workflow, with few but precisely fitting options. (What do I have to do now?)

On the other hand the current element index thing is seen more as an 'archive', with all the fancy sorting/filtering/columns/actions stuff. (Which things are in the system, and how to find old stuff.)
Hope that survives in some form.

/Werner

**Comment by wsydney76** [comment]
Just tagged a first final evaluation/test/beta/RC version (4.0.0).

Thanks for input/feedback behind the scenes, thanks to Laura&Lucy for their countless special requests...

This will now move from a side project to the core team, and therefore will become private.

Maybe someone can find some valuable ideas in it.
